file-formats/csnm/type/zif_aff_csnm_v1.intf.abap

```diff
@@ -0,0 +1,34 @@
+"! ABAP File Format for CSN Model Transport Object Generic Editor
+INTERFACE zif_aff_csnm_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">CSN File</p>
+    "! CSN file
+    BEGIN OF ty_csn_file,
+      "! <p class="shorttext">CSN File Name</p>
+      "! CSN file name
+      name TYPE string,
```

Comment:
When possible, we would even suggest 
```suggestion
      file TYPE string,
```

Reply:
Hi Katharina,
I used "name" in order to explicitly show that this the name of the file (and not its content). But I can change it

Reply:
I would agree if there was also the field "content". In this case, this is kind of reference to the file itself, so I would stick to "file".